charts/qdrant/README.md

```diff
@@ -46,6 +46,20 @@ For documentation of the settings please refer to [Qdrant Configuration File](ht
 All of these configuration options could be overwritten under config in `values.yaml`.
 A modification example is provided there.
 
+### Overrides
+You can override any value in the Qdrant configuration by setting the Helm values under the key `config`. Those settings get included verbatim in a file called `config/production.yml` which is explained further here: [Qdrant Order and Priority](https://qdrant.tech/documentation/guides/configuration/#order-and-priority)
+
+```bash
+config:
+  cluster:
+    enabled: true
+  storage:
+    snapshots_path: ./storage/snapshots
```

Comment:
I'm working on a different PR that provides better support for different snapshot paths, so I'd recommend just showing an example of setting `config.log_level: DEBUG` or something simpler

Reply:
There's an example in the values.yaml file so I just permalinked to that instead.